mava/wrappers/jumanji.py

```diff
@@ -236,16 +261,19 @@ def step(self, state: State, action: chex.Array) -> Tuple[State, TimeStep]:
             global_state=global_state,
             step_count=jnp.repeat(timestep.observation.step_count, self._env.num_agents),
         )
+        timestep.reward = jnp.sum(timestep.reward)
+
         return state, timestep
 
     def observation_spec(self) -> specs.Spec[ObservationGlobalState]:
         """Specification of the observation of the `RobotWarehouse` environment."""
+        num_actions = int(self._env.action_spec().num_values[0])
 
         agents_view = specs.Array(
             (self._env.num_agents, self._env.num_obs_features), jnp.int32, "agents_view"
         )
         action_mask = specs.BoundedArray(
-            (self._env.num_agents, 5), bool, False, True, "action_mask"
+            (self._env.num_agents, num_actions), bool, False, True, "action_mask"
```

Comment:
Nice change, but I think this catches some edge cases where we have multi-discrete action shapes

```suggestion
            self._env.action_spec().shape, bool, False, True, "action_mask"
```

Reply:
hmmmm, weird, ok then you can leave it as is and if we get an issue in the future we can deal with it then :smile: